packages/alea-frontend/pages/course-home/[courseId].tsx

```diff
@@ -190,6 +168,10 @@ const CourseHomePage: NextPage = () => {
             {t.studyBuddy}&nbsp;
             <Diversity3Icon fontSize="large" />
           </CourseComponentLink>
+          <CourseComponentLink href={`/practice-problems/${courseId}`}>
+            {<p>Practice</p>}&nbsp;
```

Comment:
Do localization of "Practice"

Reply:
Localization of all hardcoded value done.